services/httpoverrpc/client/client.go

```diff
@@ -136,16 +136,18 @@ func (p *proxyCmd) Execute(ctx context.Context, f *flag.FlagSet, args ...interfa
 			sendError(httpResp, http.StatusBadRequest, err)
 			return
 		}
-		req := &pb.LocalhostHTTPRequest{
+		req := &pb.HostHTTPRequest{
 			Request: &pb.HTTPRequest{
 				RequestUri: httpReq.RequestURI,
 				Method:     httpReq.Method,
 				Headers:    reqHeaders,
 				Body:       body,
 			},
-			Port: int32(port),
+			Port:     int32(port),
+			Protocol: "http",
+			Hostname: "localhost",
```

Comment:
should this be `httpReq.Protocol` and `httpReq.Hostname`?

Reply:
Oh you mean we should put protocol and hostname inside HTTPRequest rather than HostHTTPRequest definition. To be honest, I think we can put all fields in HostHTTPRequest to HTTPRequest like (port, hostname, protocol), since they are all parts of the http request.

Reply:
Or do you mean we should use http.Request.Host rather than a string to represent the hostname?

Reply:
My bad I misunderstood. I think we can keep this as it is.